src/components/ItaliaTheme/Header/HeaderSearch/HeaderSearch.jsx

```diff
@@ -33,6 +33,7 @@ const HeaderSearch = () => {
             setShowSearchModal(true);
           }}
           href="#search"
```

Comment:
```suggestion
          href="/search"
```
Vedi spiegazione nell'altra PR

Reply:
Fatto!